### PR TITLE
7.0.x Support XML and JSON payload in mule2bal converter

### PR DESCRIPTION
**Pull request by arukshani**

### CODE REVIEW

File: tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/builder/BallerinaASTModelBuilder.java [hunk 1]
```diff
@@ -246,7 +246,8 @@ public void endVariableRefList(int noArguments) {
     public void createAction(String actionName, boolean argsAvailable) {
         BLangModelBuilder.NameReference nameReference = nameReferenceStack.pop();
         if (processingActionInvocationStmt) {
-            modelBuilder.createActionInvocationStmt(null, null, nameReference, actionName, argsAvailable);
+            //  modelBuilder.createActionInvocationStmt(null, null, nameReference, actionName, argsAvailable);
```
Comment (milindaperera) [inline tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/builder/BallerinaASTModelBuilder.java]: let's remove commented code

Comment (arukshani) [reply tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/builder/BallerinaASTModelBuilder.java]: This was fixed in this commit >> https://github.com/wso2/product-ei/pull/979/files#diff-e926d4a6b8bdff790cea82e4ff1ce334L295

File: tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java [hunk 2]
```diff
@@ -330,4 +367,12 @@ private void createVariableWithEmptyMap(String typeOfTheParamater, String variab
         ballerinaASTAPI.createVariable(variableName, exprAvailable);
         refVarName = variableName;
     }
+
+    private String createVariableOfTypeString(String value, boolean exprAvailable) {
+        ballerinaASTAPI.addTypes(Constant.BLANG_TYPE_STRING); //type of the variable
+        ballerinaASTAPI.createStringLiteral(value);
+        String variableName = Constant.BLANG_VAR_STRING_PAYLOAD + ++variableCounter;
+        ballerinaASTAPI.createVariable(variableName, exprAvailable); //name of the variable
```
Comment (pamod) [inline tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java]: Shall we enclose the counter ?

Comment (arukshani) [reply tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java]: this is fixed in here >> https://github.com/wso2/product-ei/commit/0ed7662a6af11881f37ec6c941db1abbd454f86f

**Pull request merged**